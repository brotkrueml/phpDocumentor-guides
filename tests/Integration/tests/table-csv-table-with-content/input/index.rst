
Csv table with content
======================

..  csv-table:: Numbers
    :header: "Header 1", "Header 2"
    :widths: 15, 15

    1, "one"
    2, "two"

