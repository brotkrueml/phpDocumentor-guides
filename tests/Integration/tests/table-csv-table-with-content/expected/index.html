<!DOCTYPE html>
<html lang="en">
<head>
    <title>Csv table with content</title>

</head>
<body>
    <div class="section" id="csv-table-with-content">
    <h1>Csv table with content</h1>

            <table>
            <thead>
                            <tr>
                                            <th>Header 1</th>
                                            <th>Header 2</th>
                                    </tr>
                    </thead>
    
    <tbody>
                    <tr>
                                <td>1</td>
                                <td>one</td>
                            </tr>
                    <tr>
                                <td>2</td>
                                <td>two</td>
                            </tr>
            </tbody>
</table>

    </div>

</body>
</html>
